### DISCUSSION TITLE
Does sh1mmer unenroll the chromebook?

### DISCUSSION THREAD

**Discussion by Yashvovxb** [3kh0/ext-remover#500] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Will my admin not be able to see the enrolled chromebook? Because I dont want it to be unenrolled

**Comment by pazza2234** [comment, marked as answer]
It will unenroll it if you click enter on unenroll, otherwize you will stay enrolled. But a lot of function on your chromebook is disabled if you do not unenroll, and other then dev mode and mc chromebox (do not use on curent school chromebook, only ones from old districts or somthing) there is not mutch use for sh1mmer without unenrolling. It is relly the hole point.
